pkg/api/v1beta1/conversion.go

```diff
@@ -1054,9 +1063,149 @@ func init() {
 			if err := s.Convert(&in.Secret, &out.Secret, 0); err != nil {
 				return err
 			}
+			if err := s.Convert(&in.AWSElasticBlockStore, &out.AWSElasticBlockStore, 0); err != nil {
+				return err
+			}
+			if err := s.Convert(&in.NFSMount, &out.NFSMount, 0); err != nil {
+				return err
+			}
+			if err := s.Convert(&in.PersistentVolumeClaimVolumeSource, &out.PersistentVolumeClaimVolumeSource, 0); err != nil {
+				return err
+			}
+			return nil
+		},
+		func(in *PersistentVolume, out *newer.PersistentVolume, s conversion.Scope) error {
+			if err := s.Convert(&in.TypeMeta, &out.TypeMeta, 0); err != nil {
```

Comment:
Isn't this already covered generically with AddStructFieldConversion()

@lavalamp